packages/ui/src/theme/defaultDarkMode.ts

```diff
@@ -0,0 +1,115 @@
+import { ColorModeOverride } from './types';
+import { tokens } from './tokens';
+
+/**
+ * A basic dark mode that just flips the base color
+ * palette.
+ */
+export const defaultDarkMode: ColorModeOverride = {
```

Comment:
Since it's always passed to overrides, should it be called `defaultDarkModeOverride`?